Automattic/newspack-plugin
# feat(reader-activation): settings wizard

### All Submissions:






### Changes proposed in this Pull Request:

<img width="1281" alt="image" src="https://user-images.githubusercontent.com/820752/181345157-1a619cfe-ceb7-47e3-ad78-096fdbe2eb0c.png">

Implements basic global settings for Reader Activation features. They include:

 - Toggling reader activation entirely
 - Toggling the always-visible account link on the menu (false by default to not disrupt sites' UI on official launch)
 - Newsletter subscription text on registration for static auth modal
 - Terms and conditions text and URL, also for static auth modal



### How to test the changes in this Pull Request:

1. Check out this branch, visit the front page and confirm the account link is not visible
2. Visit the Engagement wizard and toggle the account link
3. Visit the front page and confirm the account link is rendered and working as expected
4. Change the default texts and save
5. Visit the site, open the Sign In modal and confirm the texts are applied for terms and conditions and newsletter selection while registering
6. Disable reader activation on the wizard and confirm the features become unavailable (reader registration and the blocks)

### Other information:
